src/main.py

```diff
@@ -133,8 +133,16 @@ async def handle_event(event: Dict, event_context, projects_ids: Optional[List[s
             projects_ids = await get_all_accessible_projects(context, gcp_session, token)
 
         disabled_apis = {}
+        disabled_projects = []
         for project_id in projects_ids:
             disabled_apis = {project_id: await get_all_disabled_apis(context, token, project_id)}
+            if disabled_apis[project_id] == {project_id}:
```

Comment:
Can we have check if `monitoring.googleapis.com` is in disabled_apis here? 
IMHO additional logic with passing only project_id in set is unnecessary.